docs: selective pre-rendering options (#20670)
nuxt/nuxt: docs/1.getting-started/10.deployment.md

npx nuxi generate
```

> Crawl-based pre-rendering can also be enabled in the `nuxt.config` file:

Reviewer: ```suggestion
You can enable the crawl-based pre-rendering when using `nuxt build` in the `nuxt.config` file:
```

> You can enable the crawl-based pre-rendering when using `nuxt build` in the `nuxt.config` file:

Reviewer 2: ```suggestion
You can enable crawl-based pre-rendering when using `nuxt build` in the `nuxt.config` file:
```
